model.py

```diff
@@ -5303,8 +5298,8 @@ def sort_key(edition):
         return sorted(self.identifier.primarily_identifies, key=sort_key)
 
 
+    # TODO:  policy is not used in this method.  Remove argument?
```

Comment:
I'm pretty sure we should remove this argument. I think my thinking was that when we call Metadata.apply() inside this method we should pass in a ReplacementPolicy object to control exactly what work gets done. But actually if there is a policy question, it's whether a LicensePool's presentation edition should be set/recalculated in the first place, and that policy question should be determined by whoever is calling set_presentation_edition(). 

So we might end up adding a variable to `ReplacementPolicy` which controls whether or not this method is called, but I think by the time we get in here, we're committed to completely recalculating the presentation edition.